src/js/components/Shared/TracksList.react.js

```diff
@@ -189,158 +124,224 @@ export default class TracksList extends Component {
     }
 
     selectTrack(e, id, index) {
+        if(e.button === 0 || (e.button === 2 && !this.state.selected.includes(id))) {
```

Comment:
Yea, I like this idea. It makes it a little easier to understand what's going on at a glance. I'll do it. 👍